deploy/common/deploy.sh

```diff
@@ -243,7 +245,8 @@ function setup_license() {
     echo "Injecting license ..."
     [[ -f "$license_file" ]] || { echo "License file $license_file not found!" ; return 1 ; }
 
-    local tmp="$(mktemp)"
+    tmp="$(mktemp)"
```

Comment:
This change will convert "tmp" to global variable. (Before this change, it was a local variable). Is this intended change?

Reply:
@dastgirp Makes sense to update tmp to be a local variable. Similarly refined and validated all other variable definitions to retain its original state.